configs/deploy.sh

```diff
@@ -3,26 +3,26 @@
 
 set -o errexit # exit with nonzero exit code if any line fails
 
-if [ -z "$GITHUB_TOKEN" ] && [ -z "$GITHUB_DEPLOY_KEY" ]; then
-  echo 'GITHUB_TOKEN or GITHUB_DEPLOY_KEY is not set up in Travis. Skipping deploy.'
+if [ -z "$TOKEN" ] && [ -z "$DEPLOY_KEY" ]; then
+  echo "${TOKEN_NAME} or ${DEPLOY_KEY_NAME} is not set up in ${ENV_NAME}. Skipping deploy."
```

Comment:
I just noticed that you do command substitution, i.e. `${ }`.

Can you try doing this instead: `${TOKEN_NAME}` -> `$TOKEN_NAME`. It should run the same.